test/e2e/apps/cronjob.go

```diff
@@ -20,8 +20,8 @@ import (
 	"fmt"
 	"time"
 
-	. "github.com/onsi/ginkgo"
-	. "github.com/onsi/gomega"
+	"github.com/onsi/ginkgo"
```

Comment:
Please include the lint exception from: https://github.com/kubernetes/kubernetes/blob/master/hack/.golint_failures#L636

Reply:
Thanks for your feedback. @fejta 
Now I have updated all the files in the folder `test/e2e/apps` and cleaned up `hack/.golint_failures`.